whiteboard_practice.rb

```diff
@@ -0,0 +1,53 @@
+# ## Problem #1
+# Write a method `mirror` that returns a **new array** doubling the size of a list of integers by appending the mirror image of the original sequence to the end of the list. The mirror image is the same sequence of values in reverse order. For example, if a variable called list stores this sequence of values:
+#
+# [1, 3, 2, 7]
+# and we make the following call:
+#
+# mirror(list)
+# then it should return the following values after the call:
+#
+# [1, 3, 2, 7, 7, 2, 3, 1]
+# Notice that it has been doubled in size by having the original sequence appearing in reverse order at the end of the list. You may not make assumptions about how many elements are in the list.
+
+def mirror(list)
+  i = -1
+  a = [1, 3, 2, 7]
+  b = a.dup
+
+  b.count.times do
+    a << b[i]
+    i += -1
+  end
+  return a
+end
```

Comment:
this doesn't use the parameter `list` at all...

Also what is `count`? is it the same as `.length`?